src/styles/themes/_definitions.scss

```diff
@@ -245,6 +245,16 @@ $picker-input-foreground-color: var(--picker-input-foreground-color);
 $picker-input-placeholder-color: var(--picker-input-placeholder-color);
 $picker-outline-color: var(--picker-outline-color);
 
+// Tooltip
+$tooltip-border-color-dark: var(--tooltip-border-color-dark);
+$tooltip-border-color-light: var(--tooltip-border-color-light);
+$tooltip-border-width: var(--tooltip-border-width);
+$tooltip-border-style: var(--tooltip-border-style);
```

Comment:
this feels a bit like an unnecessary abstraction. Since its already a var why bother with making it a scss var?